pandas/core/indexes/base.py

```diff
@@ -2888,6 +2890,14 @@ def union(self, other, sort=None):
         Returns
         -------
         Index
+            Returns a new Index object with all unique elements from both the original
+            Index and the `other` Index.
+
+        See Also
+        --------
+        Index.unique : Return unique values in the index.
```

Comment:
This is the same method of the docstring

Reply:
> This is the same method of the docstring

The docstring is `Index.union` and this method is `Index.unique` so I'm a little confused. Am I missing something?